day_two_guide/topics/proc_backing-up-etcd.adoc

```diff
@@ -226,8 +228,8 @@ the `/var/lib/origin/openshift.local.etcd` directory.
 +
 [IMPORTANT]
 ====
-Because etcd backup copies the whole database both v2 and v3 are included either way
-in both methods.
+Because clusters upgraded from previous versions of {product-title} might
+contain v2 data stores, back up both v2 and v3 datastores.
```

Comment:
Actually the backup copies the whole databatase (both with etcdctl2 backup and etcdctl3 snapshot copy v2 **and** v3).

So it's really only necessary to backup one of the two. The thing is that a datastore might be empty, but if it has information it **certainly** be copied.

Reply:
@juanluisvaladas, I'm not an etcd expert, but the last information that I was given is that if you're using the v2 API but have newer info, you have to manually back up the etcd database, which is why the v2 instructions contain `# cp /var/lib/etcd/member/snap/db /backup/etcd-$(date +%Y%m%d)` and why we specifically call out backing up both of them.